data/json/climbing.json

```diff
@@ -0,0 +1,236 @@
+      "msg_after": "You manage to lower yourself to the bottom."
+    },
+    "condition": { "type": "special", "flag": "ALLEY" }
+  },
+  {
+    "type": "climbing_aid",
+    "id": "furn_CLIMBABLE",
+    "//": "eg: fences, downspouts",
+    "slip_chance_mod": -5,
+    "down": { "menu_text": "Climb down %s here.", "confirm_text": "Climb down %s?", "msg_after": "You climb down." },
+    "condition": { "type": "ter_furn", "flag": "CLIMBABLE" }
+  },
+  {
+    "type": "climbing_aid",
+    "id": "furn_CLIMB_SIMPLE",
+    "//": "eg: sandbag walls, earthbag walls.  Flag is for furniture where climbing 'never fails'.",
+    "slip_chance_mod": -40,
+    "down": { "menu_text": "Clamber down onto %s here.", "confirm_text": "Climb down onto %s?", "msg_after": "You clamber down." },
+    "condition": { "type": "ter_furn", "flag": "CLIMB_SIMPLE" }
+  },
+  {
+    "type": "climbing_aid",
+    "id": "furn_LADDER",
+    "//": "currently applies to placeable stepladders, not constructed stair-like ladders",
+    "slip_chance_mod": -20,
+    "down": {
+      "menu_text": "Climb down %s here.",
+      "confirm_text": "Climb down %s?",
+      "msg_after": "You climb down the ladder.",
+      "max_height": 1
+    },
+    "condition": { "type": "ter_furn", "flag": "LADDER" }
+  },
+  {
+    "type": "climbing_aid",
+    "id": "item_stepladder",
+    "//": "We can generalize the grappling hook mechanism to portable ladders, so why not?",
+    "slip_chance_mod": -20,
+    "down": {
+      "menu_text": "Place your stepladder below and climb down.",
+      "menu_cant": "Can't place your stepladder (needs open space one story below).",
+      "menu_hotkey": "l",
+      "confirm_text": "Climb down with your stepladder?",
+      "msg_after": "You lower the ladder into place and climb down.",
+      "//": "This removes a stepladder from inventory and places it on the ground below.",
+      "max_height": 1,
+      "allow_remaining_height": false,
+      "deploy_furn": "f_ladder",
+      "easy_climb_back_up": 1
+    },
+    "condition": { "type": "item", "flag": "stepladder", "uses": 1 }
+  },
+  {
+    "type": "climbing_aid",
+    "id": "item_aluminum_stepladder",
+    "//": "We can generalize the grappling hook mechanism to portable ladders, so why not?",
+    "slip_chance_mod": -20,
+    "down": {
+      "menu_text": "Place your aluminum stepladder below and climb down.",
+      "menu_cant": "Can't place your stepladder (needs open space one story below).",
+      "menu_hotkey": "l",
+      "confirm_text": "Climb down with your stepladder?",
+      "msg_after": "You lower the ladder into place and climb down.",
+      "//": "This removes a stepladder from inventory and places it on the ground below.",
+      "max_height": 1,
+      "allow_remaining_height": false,
+      "deploy_furn": "f_aluminum_stepladder",
+      "easy_climb_back_up": 1
+    },
+    "condition": { "type": "item", "flag": "aluminum_stepladder", "uses": 1 }
+  },
+  {
```

Comment:
This climbing aid was removed from PR #68265 but I intend to add it in a followup contribution